internal/k8s/controllers/config_controller.go

```diff
@@ -133,7 +139,12 @@ func (r *ConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	}
 
 	level.Debug(r.Logger).Log("controller", "ConfigReconciler", "rendered config", dumpConfig(cfg))
+	if r.oldConfig != nil && reflect.DeepEqual(r.oldConfig, cfg) {
+		level.Error(r.Logger).Log("controller", "ConfigReconciler", "event", "configuration did not change, ignoring")
```

Comment:
Should we have level.Info here instead?

Reply:
Yep, good catch

Reply:
this could even be debug imo, unless you expect this to not show so often?

Reply:
done